stravalib/tests/unit/test_attributes.py

```diff
@@ -75,6 +77,7 @@ def test_with_incorrect_timezone(self):
         self.assertIsNone(timezone.unmarshal("(GMT+00:00) Factory"))
 
 
+@skip
```

Comment:
Skipping for now, but most of these tests will be removed, together with `attributes.py`